osu.Game/Online/Rooms/Room.cs

```diff
@@ -183,6 +188,8 @@ public void CopyFrom(Room other)
             EndDate.Value = other.EndDate.Value;
             UserScore.Value = other.UserScore.Value;
             QueueMode.Value = other.QueueMode.Value;
+            DifficultyRange.Value = other.DifficultyRange.Value;
+            PlaylistItemStats.Value = other.PlaylistItemStats.Value;
```

Comment:
Additionally `CurrentPlaylistItem` isn't getting copied across here, which I imagine is going to matter in the lounge screen.

Reply:
The whole online composite is so prone to this kind of issue :/. Not the first or last time we’ve overlooked something like this.